tron/command_context.py

```diff
@@ -199,3 +199,6 @@ def __mod__(self, _):
 
     def __nonzero__(self):
         return False
+
+    def __bool__(self):
```

Comment:
How did we get by without this before?

Reply:
i think python2 was using __nonzero__ on objects passed as `if` condition, python3 just does __bool__